src/components/navbar.js

```diff
@@ -1,24 +1,75 @@
 /** @jsx jsx */
-import { Box, Text, jsx } from 'theme-ui'
+import { Image, Flex, Box, jsx } from 'theme-ui'
 import useAuth from '../util/auth'
-import Link from './link'
+import logo from '../../static/climatedu.png'
+import NavButton from './navbutton'
+import { BsPerson } from 'react-icons/bs'
+
+const links = [
+  {
+    'location':_=>'/',
+    'text':_=>'Home'
+  },
+  {
+    'location':_=>'/about',
+    'text':_=>'About',
+    'dropdown':_=>[
+      {'location':'/about/team','text':'Our Team'},
+      {'location':'/about/mission','text':'Our Mission'}
+    ]
+  },
+  {
+    'location':_=>'/contact',
+    'text':_=>'Contact Us'
+  },
+  {
+    'location':user=>user?'/profile':'/login',
+    'text':user=>user?'Welcome '+user.displayName+'!':'Log in',
+    'icon':_=>BsPerson
+  },
+];
 
 const Navbar = () => {
-  const user = useAuth()
+  const user = useAuth();
   return (
-    <Box
+    <Flex
       sx={{
         backgroundColor: 'background',
-        mb: 4,
+        alignContent: 'flex-start',
+        alignItems: 'center',
+        flexWrap: 'wrap',
+        justifyContent: 'flex-end',
+        fontSize: '24px',
+        maxWidth: '960px',
+        margin: 'auto',
+        padding: '5px'
       }}
     >
-      this will be replaced with the real navbar
-      {user === null ? (
-        <Link to='/login'>login and register</Link>
-      ) : (
-        <Text>welcome {user.displayName}</Text>
-      )}
-    </Box>
+      <Box
+        sx={{
+          flexBasis: '7em',
+          marginRight: 'auto'
+        }}
+      >
+        <Image
+          src={logo}
+          alt='climatedu logo'
+        />
+      </Box>
+      {links.map(({location,text,icon,dropdown}) => {
+        const loc = location(user);
+        const active = window.location.pathname == loc;
```

Comment:
https://www.gatsbyjs.com/docs/gatsby-link/#add-custom-styles-for-the-currently-active-link
Please note that using `window` is not allowed and will make builds fail